grakn-engine/src/main/java/ai/grakn/engine/HttpHandler.java

```diff
@@ -91,11 +95,13 @@ protected void startCollaborators() {
 
         JacksonPrinter printer = JacksonPrinter.create();
 
-        // Start all the controllers
-        new GraqlController(factory, spark, postProcessor, printer, metricRegistry);
-        new ConceptController(factory, spark, metricRegistry);
-        new SystemController(spark, prop, factory.systemKeyspace(), graknEngineStatus, metricRegistry);
-        new CommitLogController(spark, postProcessor);
+        // Start all the DEFAULT controllers
+        new GraqlController(factory, postProcessor, printer, metricRegistry).start(spark);
+        new ConceptController(factory, metricRegistry).start(spark);
+        new SystemController(prop, factory.systemKeyspace(), graknEngineStatus, metricRegistry).start(spark);
+        new CommitLogController(postProcessor).start(spark);
```

Comment:
Maybe we should pass them in as collaborators? I guess it doesn't make much difference right now.

Reply:
I thought the same, but I figured this might be a discussion for the future, since for now we always want them

Reply:
Ya I kind of like the ides of passing in the defaults as well.